sematic/container_images.py

```diff
@@ -35,27 +34,21 @@ def get_image_uris() -> Dict[str, str]:
     -------
     The URI of the image to be used in this execution.
     """
+
     if CONTAINER_IMAGE_ENV_VAR in os.environ:
         return {DEFAULT_BASE_IMAGE_TAG: os.environ[CONTAINER_IMAGE_ENV_VAR]}
 
     tagged_uris_map = {}
 
-    dir_path, file_name = os.path.split(__main__.__file__)
-
-    image_file_regex = r"{}_(.*?)_push_at_build.uri".format(
-        os.path.splitext(file_name)[0]
-    )
-
-    for file_path in os.listdir(dir_path):
-        tag_matches = re.findall(image_file_regex, file_path)
-        if len(tag_matches) == 1:
-            tag = tag_matches[0]
-            absolute_file_path = (
-                f"{os.path.splitext(__main__.__file__)[0]}_{tag}_push_at_build.uri"
-            )
+    if CONTAINER_IMAGE_URIS_ENV_VAR in os.environ:
+        print(os.environ[CONTAINER_IMAGE_URIS_ENV_VAR])
+        image_uris = os.environ[CONTAINER_IMAGE_URIS_ENV_VAR].split('::')
 
-            with open(absolute_file_path) as f:
-                tagged_uris_map[tag] = f.read()
+        for image_uri in image_uris:
+            if not image_uri:
```

Comment:
nit: we tend to avoid relying on truthiness and instead use expressions that evaluate to bools
```suggestion
            if len(image_uri) > 0:
```

Reply:
👍